packages/build-config/sections/module-rules/webmanifest.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var path = require('path');
+
+var hopsConfig = require('hops-config');
+
+exports.default = {
+  test: /\.webmanifest$/,
+  use: [
+    {
+      loader: require.resolve('file-loader'),
+      options: {
+        name: path.join(hopsConfig.assetPath, '[name]-[hash:16].[ext]'),
```

Comment:
Should we add `emitFile: false` here too?

Reply:
Yes, please. And in node, we could probably also get rid of the `web-app-manifest-loader` itself.